docs/1000.yaml

```
@@ -606,6 +652,7 @@ Q693083:
 
 Q716171:
   title: Erdős–Ginzburg–Ziv theorem
+  decl: ZMod.erdos_ginzburg_ziv
```

I see a few `author` lines, so here are some suggestions
```suggestion
  decl: ZMod.erdos_ginzburg_ziv
  author: Yaël Dillies